client/packages/invoices/src/Prescriptions/DetailView/PrescriptionLineEdit/hooks/useDraftPrescriptionLines.tsx

```diff
@@ -28,13 +28,14 @@ export const useDraftPrescriptionLines = (
   item: DraftItem | null,
   date?: Date | null
 ): UseDraftPrescriptionLinesControl => {
-  const { id: invoiceId, status } = usePrescription.document.fields([
-    'id',
-    'status',
-  ]);
```

Comment:
I don't know why there was a hook just to pull a subset of fields from an existing data structure